### ISSUE TITLE
Is "Surjective" wrongly defined in "master" ? 

### ISSUE THREAD

**Issue by mechvel** [agda/agda-stdlib#851]
The current ``master`` has
```
module Function.Definitions.Core2 {b ℓ₂} {B : Set b} (_≈₂_ : Rel B ℓ₂) where
    ...
   Surjective : ∀ {a} {A : Set a} → (A → B) → Set (a ⊔ b ⊔ ℓ₂)
   Surjective f = ∀ x → ∃ λ y → y ≈₂ f x
```
Here ``Surjective f`` needs to mean ``(y : B) →  ∃ λ (x : A) → y ≈₂ f x``
-- which is not expressed in the code.
Am I missing something?

Also ``(x : Foo)`` often makes the code more clear than  ``∀ x``.

I have done the corresponding commit.

**Comment by MatthewDaggitt** [comment]
Yes it is *embarrassment* will fix

**Comment by andreasabel** [comment]
@mechvel wrote:
> I have done the corresponding commit.

You could submit a pull request.

**Comment by mechvel** [comment]
> You could submit a pull request.

I submitted it (done a commit)  on August 5 (836bcce)  as a part new Bin, with the main changes done to Bin.Properties.  The last my action with it was ``git push``.
The library team is supposed to review the commit and inform me, and then either merge it to 
``master`` or require some editing. This was our usual procedure this year.

**Comment by MatthewDaggitt** [comment]
Just fixed this with a62501b5 before #840 was merged in as it was getting in my way from doing other things.
